ds_judgements_public_ui/templates/includes/cards_homepage.html

```diff
@@ -1,12 +1,12 @@
 {% load static %}
 <ul class="cards-homepage">
   <li>
-    {% include "includes/card_homepage.html" with title="Recently published" body="Browse legal decisions most recently added to the Find Case Law site." link_route="search" image_src="/images/recently_added.jpg" %}
+    {% include "includes/card_homepage.html" with title="Browse by court or tribunal" body="Browse judgments and tribunals from the United Kingdoms's 18 courts and 7 tribunals." link_route="courts_and_tribunals" image_src="/images/court-crest.png" %}
```

Comment:
It would be cool to add this data to the view and loop over the cards here maybe.

Reply:
@jlhdxw this was how it was done previously I am reusing the hoemcards as this is fortesting so if this version 3 is chosen to keep and not discard then a PR can go in to do this